subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -1320,7 +1327,12 @@ static int light_ctrl_srv_start(struct bt_mesh_model *mod)
 		store(srv, FLAG_STORE_STATE);
 		break;
 	case BT_MESH_ON_POWER_UP_ON:
-		ctrl_disable(srv);
+		if (atomic_test_bit(&srv->flags,
+			FLAG_CTRL_SRV_MANUALY_ENABLED)) {
+			ctrl_enable(srv);
+		} else {
+			ctrl_disable(srv);
+		}
 		store(srv, FLAG_STORE_STATE);
 		light_set(srv,
 			  (srv->lightness->default_light ?
```

Comment:
I think this light_set call should only happen if the light_ctrl isn't manually enabled.

Reply:
Done